pdk/1.0/modules/ROOT/pages/policies-pdk-create-schema-definition.adoc

```diff
@@ -271,6 +294,52 @@ properties:
       default: {"id": "1", "name": "John Doe"}
 ----
 
+[[dataweave-expressions]]
```

Comment:
The italics might mean something earlier didn't get turned off...

Reply:
This builds correctly and looks as it should in VScode. I think this is just a github issue.